Realm/ObjectServerTests/SwiftPermissionsAPITests.swift

```diff
@@ -88,11 +88,9 @@ class SwiftPermissionsAPITests: SwiftSyncTestCase {
         var isPresent = false
         let token = results.addNotificationBlock { (error) in
             XCTAssertNil(error, "Notification returned error '\(error!)' when running test at \(file):\(line)")
-            for result in results where result == permission {
-                isPresent = results.contains(permission)
-                if (isPresent) {
-                    return
-                }
+            isPresent = results.contains(permission)
+            if isPresent {
```

Comment:
Calling `return` in a block just avoids executing the rest of the block, which is nothing in this case, so just remove the whole check:

```swift
let token = results.addNotificationBlock { error in
    XCTAssertNil(error, "Notification returned error '\(error!)' when running test at \(file):\(line)")
    isPresent = results.contains(permission)
}